Mailer/README.md

```diff
@@ -887,27 +1176,158 @@ new *email* template via API.
 | --- |---| --- | --- |
 | Authorization | Bearer *String* | yes | API token. |
 
+##### *Body:*
+
+```json5
+{
+  // optional
+  "public_listing": true // Boolean; Get only published newsletters
+}
+```
+
 ##### *Example:*
 
 ```shell
-curl -X GET \
+curl -X POST \
   http://mailer.remp.press/api/v1/mailers/mail-types \
-  -H 'Authorization: Bearer XXX'
+  -H 'Authorization: Bearer XXX' \
+  -H 'Content-Type: application/json' \
+  -d '{
+    "public_listing": true
+}'
 ```
 
 Response:
 
 ```json5
 {
-    "status": "ok",
-    "data": [
-        {
-            "id": 14,
-            "code": "alerts",
-            "title": "Breaking news"
-        }
-        // ...
-    ]
+  "2": {
+    "id": 2,
+    "code": "123",
+    "mail_type_category_id": 1,
+    "image_url": "",
+    "preview_url": "",
+    "title": "Test",
+    "description": "",
+    "locked": 0,
+    "is_multi_variant": 1,
+    "variants": {
+      "4": "Test 3",
+      "5": "Test 4"
+    }
+  },
+  "1": {
+    "id": 1,
+    "code": "demo-weekly-newsletter",
+    "mail_type_category_id": 1,
+    "image_url": null,
+    "preview_url": null,
+    "title": "DEMO Weekly newsletter",
+    "description": "Example mail list",
+    "locked": 0,
+    "is_multi_variant": 1,
+    "variants": {
+      "2": "Test 1",
+      "3": "Test 2"
+    }
+  }
+}
+```
+
+---
+
+#### POST `/api/v1/mailers/mail-type-by-code`
```

Comment:
You could probably reuse `/mailers/mail-types` API here and only add new optional filter with array of `codes` you're requesting.